lib/myaccount/README.md

```diff
@@ -16,14 +16,20 @@ okta.myAccount.profile.manage
 
 **email:**
 ```
-okta.myaccount.email.read
-okta.myaccount.email.manage
+okta.myAccount.email.read
```

Comment:
due to backend change or previous typo?

Reply:
I believe this was a typo: https://github.com/okta/okta-auth-js/blob/master/samples/generated/react-embedded-auth-with-sdk/src/config.js#L16